File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/VfsFile.java
/**
 * @file
 * @brief Vfs file object interface
 * @author vitamin.caig@gmail.com
 */

package app.zxtune.fs;

public interface VfsFile extends VfsObject {

  /**
   * @return File size in implementation-specific units
   */
  String getSize();
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/fs/VfsFile.kt
/**
 * @file
 * @brief Vfs file object interface
 * @author vitamin.caig@gmail.com
 */
package app.zxtune.fs

interface VfsFile : VfsObject {
    /**
     * @return File size in implementation-specific units
     */
    val size: String
}
